ga4gh/frontend.py

```diff
@@ -372,6 +422,57 @@ def searchVariants(version):
         version, flask.request, app.backend.searchVariants)
 
 
+@app.route('/_login', methods=['GET'])
+def login():
```

Comment:
Can we get some docstring comments on this function (and all other new externally visible functions) please? Since these are security sensitive, it needs to be really, really clear what the intended use for the function is and how it works.